Permanent/Modules/Main/Navigation/Routers/SettingsRouter.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  SettingsRouter.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 05.02.2024.
+
+import Foundation
+import SwiftUI
+
+class SettingsRouter: ObservableObject {
+    enum Page {
+        case settings
+        case account
+        case storage
+        case myArchives
+        case invitations
+        case activityFeed
+        case security
+        case legacyPlanning
+        case signUp
+    }
+    
+    var rootViewController: RootNavigationController
+    var currentView: Page
+    
+    init(rootViewController: RootNavigationController, currentView: Page) {
+        self.rootViewController = rootViewController
+        self.currentView = currentView
+    }
+    
+    func navigate(to view: Page, router: SettingsRouter) {
```

Comment:
change this to:

func navigate(to **page**: Page

instead of 

func navigate(to **view**: